storage/inmemory/accounts.go

```diff
@@ -0,0 +1,87 @@
+package inmemory
+
+import (
+	"context"
+	"fmt"
+
+	accountspb "github.com/openbank/gunk/gunk/v1/accounts"
+)
+
+type AccountStore struct {
```

Comment:
Is it possible that the store is accessed concurrently? We should add a mutex to avoid concurrent access to the map. (Or use some concurrent map implementation)